src/main/java/com/midaswebserver/midasweb/services/UserService.java

```diff
@@ -0,0 +1,92 @@
+package com.midaswebserver.midasweb.services;
+
+import com.midaswebserver.midasweb.models.User.User;
+import jakarta.servlet.http.HttpServletRequest;
+
+/**
+ * @version 0.0.1
+ * UserService is the CRUD service for Users. the majority of user logic goes through the
+ * UserServiceImp
+ * @Author Aidan Scott
+ * @since 0.0.1
+ */
+public interface UserService {
+    /**
+     * Adds user to the database
+     * If User is already in the database (same username) then the method will return false
+     *
+     * @param user
+     * @return true if user could be added, if there was a user of the same name or user is null, false
+     */
+    boolean add(User user);
+
+    /**
+     * ValidateUniqueUsername checks to see if there is another person in the database with the same username
+     *
+     * @param username
+     * @return boolean
+     */
+    boolean validateUniqueUsername(String username);
+
+    /**
+     * delete deletes a user from the database
```

Comment:
The method name in javadoc matches the method name, but the javadoc in the above method uses an upper-case for the first letter in the method.  Please be consistent (I prefer the former vs. the latter, but that's my opinion).